app/mailers/rrd_completed_mailer.rb

```diff
@@ -5,6 +5,7 @@ def build(submission)
     @submission = submission
     @disability_struct = RapidReadyForDecision::Constants.first_disability(submission) || {}
     @rrd_status = submission.rrd_status
+    @error_message = submission.form.dig('rrd_metadata', 'error') if @rrd_status == 'error'
```

Comment:
`submission.rrd_status` now returns a string so that it can return string values from `form.dig('rrd_metadata', 'offramp_reason')`.